mlir/tools/mlir-tblgen/FormatGen.h

```diff
@@ -14,7 +14,13 @@
 #ifndef MLIR_TOOLS_MLIRTBLGEN_FORMATGEN_H_
 #define MLIR_TOOLS_MLIRTBLGEN_FORMATGEN_H_
 
+#include "AttrOrTypeFormatGen.h"
+#include "FormatGen.h"
```

Comment:
Can these two be removed now? I assume the second one would also have been a recursive include?

Reply:
Yes, thanks, apologies, I should have catched that.